docs/README.md

```diff
@@ -3,101 +3,102 @@ sidebar_position: 1
 sidebar_label: Introduction
 ---
 
-# The Zed Project
+# SuperDB
 
-Zed offers a new approach to data that makes it easier to manipulate and manage
-your data.
-
-With Zed's new [super-structured data model](formats/README.md#2-zed-a-super-structured-pattern),
+SuperDB offers a new approach that makes it easier to manipulate and manage
+your data.  With its [super-structured data model](formats/README.md#2-zed-a-super-structured-pattern),
 messy JSON data can easily be given the fully-typed precision of relational tables
 without giving up JSON's uncanny ability to represent eclectic data.
 
 ## Getting Started
 
-Trying out Zed is easy: just [install](install.md) the command-line tool
-[`zq`](commands/zq.md) and run through the [zq tutorial](tutorials/zq.md).
-
-`zq` is a lot like [`jq`](https://stedolan.github.io/jq/)
-but is built from the ground up as a search and analytics engine based
-on the [Zed data model](formats/zed.md).  Since Zed data is a
-proper superset of JSON, `zq` also works natively with JSON.
+Trying out SuperDB is easy: just [install](install.md) the command-line tool
+[`super`](commands/zq.md) and run through the [tutorial](tutorials/zq.md).
 
-While `zq` and the Zed data formats are production quality, the Zed project's
-[Zed data lake](commands/zed.md) is a bit [earlier in development](commands/zed.md#status).
+Compared to putting JSON data in a relational column, the
+[super-structured data model](formats/zed.md) makes it really easy to
+mash up JSON with your relational tables.  The `super` command is a little
+like [DuckDB](https://duckdb.org/) and a little like
+[`jq`](https://stedolan.github.io/jq/) but super-structured data ties the
+two patterns together with strong typing of dynamic values.
 
-For a non-technical user, Zed is as easy to use as web search
-while for a technical user, Zed exposes its technical underpinnings
+For a non-technical user, SuperDB is as easy to use as web search
+while for a technical user, SuperDB exposes its technical underpinnings
 in a gradual slope, providing as much detail as desired,
 packaged up in the easy-to-understand
-[ZSON data format](formats/zson.md) and
-[Zed language](language/README.md).
+[Super JSON data format](formats/zson.md) and
+[SuperPipe language](language/README.md).
+
+While `super` and its accompanying data formats are production quality, the project's
+[SuperDB data lake](commands/zed.md) is a bit [earlier in development](commands/zed.md#status).
```

Comment:
I intentionally moved this paragraph to the bottom of this section. One of my challenges in this rewrite was the contrast between SuperDB-the-project (which covers everything production quality that's available right now) and "SuperDB data lake" / `super db`-the-command (which we've acknowledged will be de-emphasized in the short term... but they still invoke the words "super db"). So I want to avoid sending mixed message.

While I would not advocate amputating lake coverage from the docs (especially since we have users relying on it in production!) I'm sensing that moving lake materials to less prominent places may help reduce confusion among new readers.